yt/yt/client/api/rpc_proxy/client_impl.cpp

```diff
@@ -2244,6 +2244,20 @@ TFuture<NQueryTrackerClient::TQueryId> TClient::StartQuery(
         protoFile->set_type(static_cast<NProto::EContentType>(file->Type));
     }
 
+    for (const auto& sec : options.Secrets) {
```

Comment:
In YT, we try not to use abbreviations. Let's call it secret for C++ structure and protoSecret for protobuf field.